pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/con_pantheon_yaml_file.adoc

```diff
@@ -1,46 +1,49 @@
 [id='pantheon-yaml-file_{context}']
 
-= The Pantheon configuration file
+= The {ProductShortName} configuration file
 
-You need to create a configuration file, named `pantheon2.yml`, in the root directory of your repository, to enable a Git repository as a source that Pantheon can use to import content. This yaml file provides necessary information that Pantheon uses to interact with files in the repository.
+You need to create a configuration file, named `pantheon2.yml`, in the root directory of your repository, to enable a Git repository as a source that {ProductShortName} can use to import content. This yaml file provides necessary information that {ProductShortName} uses to interact with files in the repository.
 
 The configuration file enables the following:
 
-* Importing content from a Git repository into Pantheon
-* Uploading modules into Pantheon using the Uploader tool
+* Importing content from a Git repository into {ProductShortName}
+* Uploading modules into {ProductShortName} using the Uploader tool
 
 The configuration file defines the following:
 
-`server`:: The URL of the Pantheon server.
-`repository`:: A unique ID that Pantheon uses to reference the URL of the repository. You specify the actual URL of the repository when you synchronize the repository the first time.
-`modules`:: Patterns for modules that show up in the search results on Pantheon.
+`server`:: The URL of the {ProductShortName} server.
+`repositories`:: a multi-valued structure with the following child members:
++
+`name`:: A unique ID that {ProductShortName} uses to reference the URL of the repository. You specify the actual URL of the repository when you synchronize the repository the first time.
```

Comment:
This sounds confusing to me, since 'name' is a unique name that would show up in the user facing URL and cannot be confused with the git url.
Here is my recommendation:
`name` : A unique name that {ProductShortName} uses to reference the URL of the repository. 
Let's say for rhel-8 docs, it can be name: rhel-8-docs or any other unique string that the authors like.